wikimedia/wikimedia-wordpress-annual-report-plugin
# #982 - Stories Section

Addresses [#982](https://app.zenhub.com/workspaces/wmf-annual-report-65b16fe392e9e50d125a7ff6/issues/gh/humanmade/wikimedia/962)

To test:

- On a report page
- Insert the 'Stories' block
- Add a new slide
- Enter a name for the category and Populate the content
- Add a new slide
- Enter a name for the category, click on the 'story' inner block, associate this with a post and refresh the slide content
- Note that the slide now has auto populated block and inner-block content that you can edit
- Add multiple new slides
- View the content on the front end
- Confirm that the categories can be selected to navigation the slides
- Confirm that the left and right arrows will navigate the slides
- Confirm that where a slide has overlay content, the button triggers the content to be displayed in an overlay

View demo for more information:

https://cln.sh/Tm7d0l1C14L1PPW1rrPx
